src/routes/shortener.route.ts

```diff
@@ -1,11 +1,20 @@
 import { Router } from 'express';
-import { createUrls, getUrls, deleteUrl } from '../controllers/shortener.controller';
+import {
+  createUrls,
+  getUrls,
+  deleteUrl,
+  updateURL,
+  getSingleUrl
+} from '../controllers/shortener.controller';
 import { urlValidatorRules } from '../middleware/urlvalidator';
 import { validate } from '../middleware/validate';
 const router: Router = Router();
 
 router.post('/urls', urlValidatorRules, validate, createUrls);
 router.get('/urls', getUrls);
+router.get('/urls/:id', getSingleUrl);
+
+router.put('/urls', updateURL);
```

Comment:
Rest API design principle recommends having a route parameter for the id 

```suggestion
router.put('/urls/:id', updateURL);
```